paasta_tools/setup_kubernetes_job.py

```diff
@@ -162,19 +168,94 @@ def reconcile_kubernetes_deployment(
             kube_client=kube_client,
             application=formatted_application,
         )
+        ensure_pod_disruption_budget(
```

Comment:
Copy-paste, all these 3 `ensure_pod_disruption_budget` can be moved out of the `if` statement.

Reply:
`ensure_pod_disruption_budget` calls are still there.  Can you please either do something with them or left a comment explaining why this code need to be left unchanged?

Reply:
I've updated so there is only 1 - you were too fast :)